### ISSUE TITLE
Escape key

### ISSUE THREAD

**Issue by yvan-sraka** [VladThodo/behe-keyboard#30]
Hi, it would be nice to add an `Esc` key.
I use vim inside termux and that's the special feature that I really hope to find in a keyboard designed for programmers ;) ! Thanks

**Comment by VladThodo** [comment]
@yvan-sraka, yeah, I should have added that since the first version. I'm currently thinking where exactly it should be placed. Any suggestions? Thanks for getting in touch :-)

**Comment by anmoluppal** [comment]
@VladThodo You can get some ideas from [Hacker Keyboard](https://github.com/klausw/hackerskeyboard) layout

**Comment by victorperin** [comment]
@VladThodo thanks for the project.

Thinking about layout, could we add it after the `>` (right arrow) on the arrows layout?

**Comment by VladThodo** [comment]
@victorperin, I'm actually thinking about replacing the existing 'OK' key with ESC. There's already an entet key on the arrows layout, thus the OK is kinda redudant in this case.

**Comment by victorperin** [comment]
I Agree, didn't bring that up because I thought it might have some specific use cases.
